code/src/utest/pcgen/base/formula/AddingFormulaTest.java

```diff
@@ -55,30 +59,34 @@ public void testEquality()
 		assertFalse(f1.equals(f4));
 	}
 
+	@Test
 	public void testPositive()
 	{
 		AddingFormula f = new AddingFormula(3);
-		assertEquals(8, f.resolve(Integer.valueOf(5)).intValue());
-		assertEquals(10, f.resolve(Double.valueOf(7.5)).intValue());
+		assertEquals(8, f.resolve(5).intValue());
+		assertEquals(10, f.resolve(7.5).intValue());
 		testBrokenCalls(f);
 	}
 
+	@Test
 	public void testZero()
 	{
 		AddingFormula f = new AddingFormula(0);
-		assertEquals(5, f.resolve(Integer.valueOf(5)).intValue());
-		assertEquals(2, f.resolve(Double.valueOf(2.3)).intValue());
+		assertEquals(5, f.resolve(5).intValue());
+		assertEquals(2, f.resolve(2.3).intValue());
 		testBrokenCalls(f);
 	}
 
+	@Test
 	public void testNegative()
 	{
 		AddingFormula f = new AddingFormula(-2);
-		assertEquals(3, f.resolve(Integer.valueOf(5)).intValue());
-		assertEquals(-8, f.resolve(Double.valueOf(-6.7)).intValue());
+		assertEquals(3, f.resolve(5).intValue());
+		assertEquals(-8, f.resolve(-6.7).intValue());
 		testBrokenCalls(f);
 	}
 
+	@Test
```

Comment:
I don't think this should be a test... it's a utility function and probably should be renamed to not start with "test"